.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.15.x, 1.16.x]
+        go-version: [1.15.x, 1.16.x, 1.17.x]
```

Comment:
Dockerfile needs to be update https://github.com/opencontainers/runc/blob/55e93b89f185631a87d87b9f4e7e08437357eb2c/Dockerfile#L1

Reply:
Can we drop support for Go 1.15?

Reply:
> Dockerfile needs to be update

done

> Can we drop support for Go 1.15?

I'd say let's have it for some time as it just became unsupported a day ago.

This is also a separate issue; I'd like to add 1.17 now and drop 1.15 later and separately.